torch/distributed/rpc.py

```diff
@@ -96,6 +99,14 @@ def init_rpc(name, backend='pg'):
         raise RuntimeError("Unrecognized RPC backend ", backend)
 
 
+def get_id(workerName=None):
```

Comment:
nit: call this `get_worker_id` and add docs?

Reply:
agree. I'd prefer always make names descriptive.
Later, we will also have ref_id, fork_id.